### PR TITLE
Disabling deprecated functions in GenModel

### PR DESCRIPTION
**Pull request by st-pasha**
(as discussed with @tomk)

---
This change is [Reviewable](https://reviewable.io/reviews/h2oai/h2o-3/520)

### CODE REVIEW

**Comment by tomkraljevic** [comment]
@st-pasha: it looks ok to me.
should have @mmalohlava look as well.

**Comment by st-pasha** [comment]
@h2o-ops please test!

**Comment by abal5** [comment]
@h2o-ops please test

File: h2o-genmodel/src/main/java/water/genmodel/IGeneratedModel.java [hunk 1]
```diff
@@ -73,16 +73,16 @@
...
+     */
 
     /** Predict the given row and return prediction using given number of iterations (e.g., number of trees from forest).
      *
      * @param data row holding the data. Ordering should follow ordering of columns returned by getNames()
      * @param preds allocated array to hold a prediction
      * @param maxIters maximum number of iterations to use during predicting process
      * @return returned preds parameter filled by prediction
-     */
     public float[] predict(double[] data, float[] preds, int maxIters);
```
Comment (st-pasha) [inline h2o-genmodel/src/main/java/water/genmodel/IGeneratedModel.java]: Yes, the corresponding API call was the following:
```
@Override
public float[] predict(double[] data, float[] preds, int maxIters) {
  throw new UnsupportedOperationException("Unsupported operation - use score0 method!");
}
```

Comment (mmalohlava) [inline h2o-genmodel/src/main/java/water/genmodel/IGeneratedModel.java]: oki, it was used before by one customer but probably dead now.

**Pull request merged**